misc/userscripts/password_fill

```diff
@@ -209,6 +210,81 @@ choose_entry_zenity_radio() {
     choose_entry_menu || true
 }
 
+# Default backend selection.
+backend="pass"
+
+# =======================================================
+# backend: LastPass. 
+# The LastPass backend depnds only on a system installation of lastpass-cli.
+# Users must define their lastpass username in password_fill_rc and set 
+# the backend variable to "last_pass"
+# e.g.
+#
+# #!/usr/bin/env bash
+# username=lp-username@email.com
+# backend=last_pass
+#
+# Temporary files are created that store account user name along
+# with the lastpass id for a given address. Users can redefine where the
+# temporary files will be saved with LP_CACHE_PREFIX the default is as such.
+#
+# No password data is ever written to a file with this method.
+
+LP_CACHE_PREFIX=$HOME/.config/qutebrowser/id_cache
+
+last_pass_backend() {
```

Comment:
I think it should be `lastpass_backend` because the package on archlinux and debian seems to be `lastpass-cli` and you are also calling it `LastPass` (without a `-` or `_`) in the comment above.

I'm not sure whether it is (cryptographically) reasonable that you extract parts of the password store entry (the username) to a plaintext file in some temporary directory. (I probably wouldn't want my password store to do that).